docs-ui/components/_activity.stories.js

```diff
@@ -17,7 +17,7 @@ const user = {
 };
 
 export default {
-  title: 'UI/Activity/Item',
+  title: 'Core/_Activity/Item',
```

Comment:
I know this is a crime but it shouldn’t be the first thing you see in these docs and should be in a separate section. For now: it stays.

Reply:
Only adding it here because Storybook picks which item is first in the left hand nav based on what the name of the file is in /components/

Reply:
Since you renamed the file shouldn't this be in UI?